openstack_controller/tests/metrics.py

```diff
@@ -157,6 +157,20 @@
 
 NEUTRON_PROJECT_METRICS = NEUTRON_NETWORK_METRICS | NEUTRON_QUOTA_METRICS
 
+VOLUMES_METRICS_CINDER = [
```

Comment:
this doesn't seem to be used anywhere

Reply:
Yes, this isn't needed, and I have removed it.